### PR TITLE
create sample data fields on frontend, and save the filled out sample data to the db by the backend

### LINKED ISSUE #50: Survey Page to answer questions
component -> type 
Short 
long 
number 
bool

### PR DESCRIPTION
**Pull request by gerigol**

### CODE REVIEW

File: src/frontend/src/component/QuestionTypeDDMenu.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { Select } from 'antd';
```
Comment (MateWagner) [inline src/frontend/src/component/QuestionTypeDDMenu.js]: if we use this then please rename it to QuestionTypeSelect.js

File: src/frontend/src/pages/sampledata/SampleData.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+  const onFinish = async (values) => {
+    loadingMessage();
+   
+    const valuesArray = Object.entries(values).map(([key, value]) => ({ [key]: value }));
+
+    const option = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify(valuesArray, id, formID),
```
Comment (MateWagner) [inline src/frontend/src/pages/sampledata/SampleData.js]: need object

File: src/frontend/src/pages/sampledata/SampleData.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
+import { useParams } from "react-router-dom";
+import { Form, Button, message } from "antd";
+import { QuestionItem } from "./QuestionItem";
+import useAuth from "../../hook/useAuth";
+import useDownFetch from "../../hook/useDownFetch";
+import upFetch from "../../api/upFetch";
+
+const PATH = "api/v1/question/getQuestions/";
```
Comment (MateWagner) [inline src/frontend/src/pages/sampledata/SampleData.js]: api/v1/question/by-form-id/ or something like that

**Review by MateWagner — CHANGES_REQUESTED** [review]

File: src/frontend/src/pages/sampledata/SampleData.js (hunk 3 above)
Comment (gerigol) [reply src/frontend/src/pages/sampledata/SampleData.js]: changed to: /api/v1/question - this is now the current url of the endpoint.

**Review by MateWagner — DISMISSED** [review]

**Pull request merged**